src/SourceBuild/content/repo-projects/fsharp.proj

```diff
@@ -19,9 +19,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <RepositoryReference Include="arcade" />
```

Comment:
Add this back in since it's top-level.

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/608e3adab65aca27adcc828a17383c6b401466dc